.github/workflows/channels-ci.yml

```diff
@@ -93,8 +108,9 @@ jobs:
         # if: steps.npm-cache.outputs.cache-hit != 'true'
         run: |
           make node_modules
-          make e2e/playwright/node_modules
+          # make channels/e2e/playwright/node_modules
       - name: ci/lint
+        working-directory: webapp/channels
```

Comment:
I guess this means we only lint the channels app? Not something we have to solve right now, but we should probably decide on whether we want to have product distinctions at this layer or not.

Reply:
Yeah, that's right. I do want to get every part of the web app covered by this, but I focused on the stuff from the web app repo first